src/coreclr/jit/compiler.cpp

```diff
@@ -4882,6 +4884,11 @@ void Compiler::compCompile(void** methodCodePtr, ULONG* methodCodeSize, JitFlags
                 DoPhase(this, PHASE_VN_COPY_PROP, &Compiler::optVnCopyProp);
             }
 
+            if (doBranchOpt)
```

Comment:
Earlier, we would do redundant branch elimination after CSE and now we do it before. Is that the source of diff?

Reply:
Yes